zowex/src/main.rs

```diff
@@ -129,6 +142,75 @@ fn main() -> std::io::Result<()> {
     }
 }
 
+/**
+ * Determine if the command to run is a "zowe daemon" command that we cannot
+ * run from our executable. For such commands, display a zowe NodeJs command
+ * that can be run instead, and exit the current command.
+ *
+ * @param cmd_line_args
+ *      The user-supplied command line arguments to the zowe command.
+ *      Each argument is in its own vector element.
+ */
+fn exit_when_alt_cmd_needed(cmd_line_args: &Vec<String>) {
+    // commands other than daemon commands can be run by our exe
+    if cmd_line_args.len() < 2 {
+        return;
+    }
+    if cmd_line_args[0] != "daemon" {
+        return;
+    }
+    if cmd_line_args[1] != "enable"  &&  cmd_line_args[1] != "disable"  {
+        return;
+    }
+
+    // we can run any of the help requests for the daemon commands
+    if cmd_line_args.len() >= 3 {
+        let third_parm: &str = cmd_line_args[2].as_str();
+        match third_parm {
```

Comment:
Are we only looking at the 3rd parameter ?
should we look in the entire array `if len() >= 3` ?
ex: `zowe zosmf check status --host asdf --help --port 123`

Reply:
I think you may be overlooking the logic of the previous code block. If the command is not "zowe daemon enable-or-disable", the function returns (which passes the command to the daemon). We only get to the block that you identified if we already know that this is a daemon command. The intent of the remainder of the function is to state that the EXE cannot run daemon commands and then exits the EXE.  However, if we already know it is a daemon command, a daemon command asking for help can still be passed to the daemon. 

```
// will be sent to daemon because it is a daemon command and parm[3] is --help
zowe daemon enable --help

// will be sent to daemon because it is NOT a daemon command and never reaches the test for parm[3]
zowe zosmf check status --host asdf --help --port 123
```

The short answer is that both conditions work and display the appropriate help.

Reply:
(－‸ლ)
Makes sense! 😋